deployments/terraform/modules/consumer/variables.tf

```diff
@@ -0,0 +1,40 @@
+variable "project" {}
+variable "environment" {}
+variable "family" {}
+variable "application" {}
+
+variable "alarm_create" {
```

Comment:
if the module is alarm context only, we can skip the alarm prefix for the variables i guess.